src/services/edge-firewall-rules-engine-services/load-edge-firewall-rules-engine-service.js

```diff
@@ -11,6 +11,29 @@ export const loadEdgeFirewallRulesEngineService = async ({ id, edgeFirewallId })
   return parseHttpResponse(httpResponse)
 }
 
+/**
+ * Parse each criteria based on its type
+ * @param {Array} criterias
+ * @returns {Array}
+ */
+const parseCriterias = (criterias) => {
```

Comment:
Drop the "S". `Criteria` is already plural

<img width="691" alt="Screenshot 2024-08-15 at 9 42 03 AM" src="https://github.com/user-attachments/assets/2ad546cc-6c2b-4470-b3ec-e51cecb9b55e">

Reply:
done